<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientEditView</class>
 <widget class="QWidget" name="PatientEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑患者信息</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>68</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QLineEdit" name="dbEditName"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>出生日期：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="dbComboSex">
           <item>
            <property name="text">
             <string>男</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>女</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>姓名：</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="dbEditMobile"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>性别：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="dbEditID"/>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="dbCreateTimeStamp"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>体重</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateEdit" name="dbDataEditDOB"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>身高：</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>手机号：</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="dbSpinHeight"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>身份证：</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="dbSpinWeight"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="dbEditIDCard"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>创建时间：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>68</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dbEditID</tabstop>
  <tabstop>dbEditName</tabstop>
  <tabstop>dbEditIDCard</tabstop>
  <tabstop>dbComboSex</tabstop>
  <tabstop>dbDataEditDOB</tabstop>
  <tabstop>dbSpinHeight</tabstop>
  <tabstop>dbSpinWeight</tabstop>
  <tabstop>dbEditMobile</tabstop>
  <tabstop>dbCreateTimeStamp</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
